drivers/dac/dac_esp32.c

```diff
@@ -0,0 +1,131 @@
+/*
+ * Copyright (c) 2022 Espressif Systems (Shanghai) Co., Ltd.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define DT_DRV_COMPAT espressif_esp32_dac
+
+#include <soc.h>
+#include <zephyr/device.h>
+#include <zephyr/drivers/dac.h>
+#include <zephyr/drivers/clock_control.h>
+#include <hal/rtc_io_types.h>
+#include <hal/rtc_io_hal.h>
+#include <hal/rtc_io_ll.h>
+#include <hal/dac_hal.h>
+#include "driver/dac_common.h"
+
+#include <esp_attr.h>
```

Comment:
Nitpick: I don't see any function that currently uses IRAM_ATTR or similar, so what is the usage of this include file?